src/Employer/Employer.Web/Views/Shared/_VacancySummaryDisplayPartial.cshtml

```diff
@@ -22,7 +22,8 @@
     </div>
     <div asp-condition="Model.IsDisabilityConfident" class="grid-row" >
         <div class="column-two-thirds small-bottom-margin">
-            <img src="/img/disability-confident_545x172.png" class="disability-confident-logo" alt="Employer is registered as disability confident">
+            <img src="/img/disability-confident_545x172.png" class="disability-confident-logo" alt="A logo confirming that the employer is registered as disability confident">
+            <span class="visuallyhidden">This vacancy meets disability confident requirements</span>
```

Comment:
Does this visually hidden classed element not have to appear on the preview pages as well?

Reply:
👍